RELEASING.md

```diff
@@ -0,0 +1,99 @@
+# Release process
+
+Only for Maintainers.
+
+1.Tag with version to be released e.g.:
+
+   ```sh
+   git tag -a 0.4.0-beta -m "0.4.0-beta"
+   git push origin 0.4.0-beta
+   ```
+
+2.Do Draft Github release. Run the following PowerShell from the root of the
+   repo.
+
+```powershell
+    $changelogs = Get-ChildItem -Path . -Recurse -Filter changelog.md
+    foreach ($changelog in $changelogs)
+    {
+     Add-Content -Path .\combinedchangelog.md $changelog.Directory.Name
+     $lines = Get-Content -Path $changelog.FullName
+     $started = $false
+     $ended = $false
+     foreach ($line in $lines)
+         {
+            if($line -like "## *" -and $started -ne $true)
+            {
+              $started = $true
+            }
+            elseif($line -like "## *" -and $started -eq $true)
+            {
+              $ended = $true
+              break
+            }
+            else
+            {
+                if ($started -eq $true)
+                {
+                    Add-Content -Path .\combinedchangelog.md $line
+                }
+            }
+         }
+    }
+```
+
+   This generates combined changelog to be used in Github release.
+
+3.Run the following PowerShell script from the root of the repo.
+   This updates all the changelog to have release date for the
+   current version being released.
+   Replace the date with actual date, version with actual version.
+   The actual version would be the tag name from step1 appended with
+   ".1"
+
+```powershell
+    $changelogs = Get-ChildItem -Path . -Recurse -Filter changelog.md
+    foreach ($changelog in $changelogs)
+    {
+     (Get-Content -Path $changelog.FullName) -replace "Unreleased", "Unreleased
+
+    ## 0.6.0-beta.1
```

Comment:
theoretically you can take it from the current tag on the branch. Not sure if it worth scripting it